docpages/install/install-vcpkg.md

```diff
@@ -12,7 +12,7 @@ To install D++ on a system from VCPKG:
 c:\vcpkg>vcpkg list dpp
 dpp:x64-windows                                    10.0.15          D++ Extremely Lightweight C++ Discord Library.
 ```
-- You may now use the library within a `cmake` based project by adding instructions such as these to your `CmakeLists.txt`:
+- You may now use the library within a `cmake` based project by adding instructions such as these to your `CMakeLists.txt`:
```

Comment:
I would also suggest capitalising `cmake` to `CMake`, so it follows the actual name :)

Reply:
i think this depends - `cmake` is the command and CMake is the program/project name

Reply:
Yes, however, in this context it references the program name itself "cmake based project".